test/ingest_search/ingest_search_test.go

```diff
@@ -0,0 +1,233 @@
+
+				// Get SID results
+				searchResultsResp, resErr := testenv.GetSearchResults(podName, sid, deployment)
+				if resErr != nil {
+					testenvInstance.Log.Error(resErr, "Failed to get search results on pod", "pod", podName, "sid", sid)
+					return splcommon.PhaseError
+				}
+
+				// Display results for debug purposes
+				prettyResults, jsonErr := json.MarshalIndent(searchResultsResp, "", "    ")
+				if jsonErr != nil {
+					testenvInstance.Log.Info("Failed to generate pretty json", jsonErr)
+				} else {
+					testenvInstance.Log.Info("Search results:", "prettyResults", string(prettyResults))
+				}
+
+				return standalone.Status.Phase
+			}, deployment.GetTimeout(), PollInterval).Should(Equal(splcommon.PhaseReady))
+		})
+	})
+
+	Context("Standalone deployment (S1)", func() {
+		It("can ingest custom data to new index and search", func() {
+
+			standalone, err := deployment.DeployStandalone(deployment.GetName())
+			Expect(err).To(Succeed(), "Unable to deploy standalone instance ")
+
+			// Wait for standalone to be in READY Status
+			testenv.StandaloneReady(deployment, deployment.GetName(), standalone, testenvInstance)
+
+			// Verify splunk status is up
+			Eventually(func() splcommon.Phase {
+				podName := fmt.Sprintf("splunk-%s-standalone-0", deployment.GetName())
+
+				splunkBin := "/opt/splunk/bin/splunk"
+				username := "admin"
+				password := "$(cat /mnt/splunk-secrets/password)"
+				splunkCmd := "status"
+
+				statusCmd := fmt.Sprintf("%s %s -auth %s:%s", splunkBin, splunkCmd, username, password)
+				command := []string{"/bin/bash"}
+				statusCmdResp, stderr, err := deployment.PodExecCommand(podName, command, statusCmd, false)
+				if err != nil {
+					testenvInstance.Log.Error(err, "Failed to execute command on pod", "pod", podName, "statusCmd", statusCmd, "statusCmdResp", statusCmdResp, "stderr", stderr)
+					return splcommon.PhaseError
+				}
+
+				if !strings.Contains(strings.ToLower(statusCmdResp), strings.ToLower("splunkd is running")) {
+					testenvInstance.Log.Error(err, "Failed to find splunkd running", "pod", podName, "statusCmdResp", statusCmdResp)
+					return splcommon.PhaseError
+				}
+
+				testenvInstance.Log.Info("Waiting for standalone splunkd status to be ready", "instance", standalone.ObjectMeta.Name, "Phase", standalone.Status.Phase)
+				return standalone.Status.Phase
+			}, deployment.GetTimeout(), PollInterval).Should(Equal(splcommon.PhaseReady))
+
+			// Create an index
+			podName := fmt.Sprintf("splunk-%s-standalone-0", deployment.GetName())
+			indexName := "myTestIndex"
+
+			// Create an index on a standalone instance
+			err = testenv.CreateAnIndexStandalone(indexName, podName, deployment)
+			Expect(err).To(Succeed(), "Failed response to add index to splunk")
+
+			testenvInstance.Log.Info("Added index to Splunk", "instance", standalone.ObjectMeta.Name, "indexName", indexName, "Phase", standalone.Status.Phase)
```

Comment:
nit: Looks like a bit of duplicate log, similar to one on line 98?

Reply:
We can remove this since the `CreateAnIndexStandalone()` prints a success log as well.